spec/models/guide_spec.rb

```diff
@@ -98,6 +98,30 @@
     end
   end
 
+  describe 'fields inheritance' do
+    let!(:guide) do
+      create(:guide,
+        extra: 'guide extra',
+        expectations: [{binding: 'guide', inspection: 'Uses:expectations'}],
+        custom_expectations: 'expect: declares foo;',
+        exercises: [exercise])
+    end
+
+    let(:exercise) do
+      build(:exercise,
+        extra: 'foo exercise extra',
+        expectations: [{binding: 'foo', inspection: 'Not:Uses:foo'}],
+        custom_expectations: 'expect: assigns foo;')
+    end
+
+    it { expect(guide.raw_expectations).to eq guide.raw_expectations }
```

Comment:
I think you meant this? Or am I missing something?

```suggestion
    it { expect(guide.raw_expectations).to eq [{binding: 'guide', inspection: 'Uses:expectations'}]}
```

Reply:
:+1: